### ISSUE TITLE
settings to always mute mic when activated

### ISSUE THREAD

**Issue by wbolster** [wbolster/nothing-to-say#2]
...to ensure that you always join conference calls in a muted state.

**Comment by wbolster** [comment]
depends on #1

**Comment by semente** [comment]
Is the idea add a push-to-talk functionality? If so, I recommend make this optionally.

I would suggest you to set as muted when starting a GNOME session. If the user change the state (muted/unmuted) it persists only during a session, and when using push-to-talk (pressing and holding mouse button on the icon or a shortcut) it will set the microphone to unmuted always when the key/button is released.

**Comment by wbolster** [comment]
push to talk already works with the keyboard shortcut. this issue is about making a preference option to mute the microphone whenever an application starts recording. i can imagine not everyone wants that, hence the preference option.

**Comment by semente** [comment]
Oh, I see. I didn't install this feature for the push-to-talk functionality but just to have a indicator on the top bar telling the state of my mic. I were also looking for a extension that sets the mic to mute when a start a GNOME session.

Just saw the behavior of push-to-talk shortcut. I like it. Maybe also allow it by pressing and holding the mouse button on the indicator's icon.

But anyway, what do you think about add a option to set the mic to mute when starting a GNOME session?

Thanks!

**Comment by wbolster** [comment]
agreed. in addition to the flag to control whether to mute when activated, there should also be a flag to mute i startup/login, and after recording finished. whether this should be one, two, or three config flags is open to debate. :)
